ci/docker/linux.Dockerfile

```diff
@@ -19,5 +21,4 @@ COPY ./ ./worker/
 
 WORKDIR /build/worker
 
-COPY ci/docker/entrypoint.sh ./entrypoint.sh
-ENTRYPOINT ["./entrypoint.sh"]
+ENTRYPOINT [ "/bin/bash" ]
```

Comment:
Similar to the other PR we can do:

```
ENTRYPOINT [ "/bin/bash", "ci/docker/entrypoint.sh" ] 
```

then just append the script we want to run in the docker-compose declaration